background-wrapper.js

```diff
@@ -0,0 +1,23 @@
+import * as background from '/background.js'
```

Comment:
Why do we need the wrapper?

Reply:
the wrapper is able to catch thrown errors any time anything invoked background.sdatortautoplay